### DISCUSSION TITLE
Is There a Way to Re-arrange Specific Elements like Pagination and Search?

### DISCUSSION THREAD

**Discussion by batibot323** [l-lin/angular-datatables#1518] [Q&A] [answered]
Hi! I've used angular-datatables for a long time however I can't find a way to re-arrange the pagination controls. I've managed to change its styling however I can't move it from the bottom of table to the upper-right of it. I'm trying to look into this [page](https://datatables.net/examples/basic_init/dom.html) but I think I'd need jQuery to do that.

Is there a way to do it the Angular way and that is more in line with the implementation from the [newer site](https://l-lin.github.io/angular-datatables)?

*Edit*:
Okay, I just had a major breakthrough. This [old docu about 'dom'](https://datatables.net/reference/option/dom#Examples) is very useful. This is what I have now:
```javascript
this.dtOptions = {
  pageLength: 10,
  dom: '<ip<t>l>'
};
```

**Comment by l-lin** [comment, marked as answer]
Answered by @batibot323: 

> Okay, I just had a major breakthrough. This old docu about 'dom' is very useful. This is what I have now:
> 
> ```javascript
> this.dtOptions = {
>   pageLength: 10,
>   dom: '<ip<t>l>'
> };
> ```
